specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -344,23 +352,35 @@ Similarly, if the app registers multiple times for startup, this is also
+| static void<br/>RegisterForFileTypeActivation(<br/> string supportedFileTypes[],<br/> string logo,<br/> int index,<br/> string exePath<br/>); |                                                              |
+| static void<br/>RegisterForFileTypeActivation(<br/> string supportedFileTypes[],<br/> string logo,<br/> int index,<br/> string supportedVerbs[]<br/>); |                                                              |
+| static void<br/>RegisterForFileTypeActivation(<br/> string supportedFileTypes[],<br/> string logo,<br/> int index<br/>); |                                                              |
+| static void RegisterForFileTypeActivation(<br/> string supportedFileTypes[],<br/> string exePath<br/>); |                                                              |
+| static void RegisterForFileTypeActivation(<br/> string supportedFileTypes[]<br/>); |                                                              |
+| static void RegisterForProtocolActivation(<br/> string name,<br/> string exePath<br/>); | **name**:<br/>the protocol identifier, eg "foo" or "https".<br/> **exePath**: path to the<br/>executable to be activated. |
+| static void RegisterForProtocolActivation(<br/> string name<br/> ); |                                                              |
+| static void RegisterForStartupActivation(<br/><br/> string taskId,<br/> bool isDisabled,<br/> string displayName,<br/> string exePath); | **taskId**: a string identifier which the app can use in<br/>the StartupTask API to request enabling or disabling of the startup<br/>behavior.<br/> **isDisabled**: the startup behavior defaults to isDisabled=true. Unpackaged apps can enable the behavior through this API. Centennial apps can also enable it,<br/> so long as the user has activated the app at least once. UWP apps cannot directly enable this - the user has to go to Settings to enable it. These are the 3 existing behaviors for the different app types.<br/> **displayName**: used in the Settings -<br/>Apps - Startup page.<br/> **exePath**: path to the executable to be activated. |
```

Comment:
DisplayName is not controllable through the API.  The platform takes it from the information embedded in the version resource data for the .EXE target.  We need to remove this from the parameters.

Reply:
Fixed.